gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PagerDutyController.groovy

```diff
@@ -39,7 +40,7 @@ class PagerDutyController {
   @Autowired
   PagerDutyService pagerDutyService
 
-  @RequestMapping("/services")
+  @RequestMapping(value = "/services", method = RequestMethod.GET)
```

Comment:
I'd use the new `@GetMapping` instead, like we do [here](https://github.com/spinnaker/gate/blob/master/gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ManagedController.java#L57)

Reply:
sure, updated the files i touched